firebase.json

```diff
@@ -4,11 +4,22 @@
     "indexes": "firestore.indexes.json"
   },
   "hosting": {
-    "public": "public",
+    "public": "build",
     "ignore": [
       "firebase.json",
       "**/.*",
       "**/node_modules/**"
+    ],
+    "rewrites": [
+      {
+        "source": "**",
+        "destination": "/index.html"
+      }
+    ]
+  },
+  "functions": {
+    "predeploy": [
+      "npm --prefix \"$RESOURCE_DIR\" run lint"
```

Comment:
the linter setup could have been a different PR ;-)

Reply:
Sorry, it's automatically added by cloud functions init. I will split it :)

Reply:
I've added additional PR only for that files. I'll rebase this branch as soon as the "init cloud functions" is merged into master.

Reply:
Done.